bower.json

```diff
@@ -1,25 +1,25 @@
 {
   "name": "ember-frost-bunsen",
   "dependencies": {
-    "ember": "2.5.0",
+    "ember": "^2.6.0",
```

Comment:
So, we want to force all consumers to use Ember `2.6` or later? Are there changes here that require `2.6`? Or would it be safe to just make this `^2.5.0` so it still works with apps that are pinned at `2.5.0`?